core/src/main/java/apoc/result/VirtualNode.java

```diff
@@ -39,14 +39,18 @@ public VirtualNode(long nodeId) {
     }
 
     public VirtualNode(Node node, List<String> propertyNames) {
-        this.id = node.getId();
+        this(node, propertyNames, true);
+    }
+
+    public VirtualNode(Node node, List<String> propertyNames, boolean withTrueId) {
+        this.id = withTrueId ? node.getId() : - node.getId() - 1;
```

Comment:
prova così:
```suggestion
    public VirtualNode(Node node, List<String> propertyNames) {
        this.id = -node.getId();
```

Reply:
Ho messo questo booleano per distinguere dove è utilizzato nella `apoc.create.*` da dove è utilizzato in altre procedure, ad esempio nelle `apoc.npl.*`, perché in queste ultime se faccio il negativo del nodo mi falliscono diverse procedure, ad esempio nella `AWSProceduresAPIWithDummyClientTest`.
Visto che comunque, in quest'ultimo caso queste procedure danno la possibilità di persistere i risultati tramite `write:true`, mi serve l'id del nodo di partenza per creare le relazioni, ad esempio in questo caso l'id del nodo `(a:Article {uri: "https://neo4j.com/blog/pokegraph-gotta-graph-em-all/"})` https://neo4j.com/labs/apoc/4.1/overview/apoc.nlp/apoc.nlp.aws.entities.graph/#usage-apoc.nlp.aws.entities.graph


Ps: se è ok, lo aggiungo come commento nel codice